BoardConfigPlatform.mk

```diff
@@ -100,7 +100,7 @@ BOARD_USES_METADATA_PARTITION := true
 
 # SUPER PARTITION
 BOARD_ASUS_DYNAMIC_PARTITIONS_PARTITION_LIST := odm product system system_ext vendor
-BOARD_ASUS_DYNAMIC_PARTITIONS_SIZE := 7513755648
+BOARD_ASUS_DYNAMIC_PARTITIONS_SIZE := 3756877824
```

Comment:
Could you document this here and add why?

Reply:
yes, will do this later tonight

Reply:
Done, should be self-explanatory now